### ISSUE TITLE
Question about the density of Point Cloud

### ISSUE THREAD

**Issue by junhua-l** [yizhangphd/FreqPCR#1]
Hi, thank you for your great work. 

In A.1. End-to-end Rendering Pipeline, Rasterization Radius, I notice a sentence: "In practice, the selection of the radius depends on the density of point cloud, and we can adjust the radius by observing the depth map obtained by rasterization. " 

Does it mean that I can change the density of an object's point cloud after preprocessing, s.t, 90%, 70%, 50%, as long as I change the size of rasterization radius accordingly? 

SIncerely,

**Comment by seanywang0408** [comment]
Not exactly. This sentence means that if you want to tune the radius (which would affect the rendering quality), you can look at the depth map obtained by rasterization. A best radius would be that most pixels in depth map are occupied. If you want to change the density of point clouds after preprocessing, you could change the ``thres`` parameter in the config file.

**Comment by junhua-l** [comment]
I see. Changing point cloud density is easy. I want to make it suitable for different point cloud densities. For example, if I manually change the same point cloud into different densities, s.t, s.t, 90%, 70%, 50%, would the rendering algorithm still perform satisfactorily? Could you give me some suggestions on what parameters to change in the config file or what I should do for different densities?

Thank you

**Comment by seanywang0408** [comment]
The density of point clouds would definitely influence the performance of algorithms. Theoretically, the higher density point clouds has, the better the algorithm performs. However, high density also leads to heavy computation and memory cost. If you want to improve performance when the point cloud is in low density, I would suggest to enlarge the rasterization radius. It could alleviate the issue brought by low density.
